utils/build-presets.ini

```
@@ -2400,7 +2400,7 @@ native-llvm-tools-path=%(toolchain_path)s
 native-clang-tools-path=%(toolchain_path)s
 
 build-ninja
-lit-args=--filter=stdlib/ -v
+lit-args=--filter=^stdlib/ -v
```

I played about with this locally, and it seems lit test filters match against the entire string for e.g:

```
Swift-validation(macosx-x86_64) :: stdlib/SetAlgebra.swift
```

So I *believe* the filter we want here is actually something like `:: stdlib/`.

```suggestion
lit-args=--filter=':: stdlib/' -v
```

---

Done :) Thanks